packages/tools/devtools/devtools-core/src/data-visualization/DataEditing.ts

```diff
@@ -22,8 +22,9 @@ export type EditSharedObject = (sharedObject: ISharedObject, edit: Edit) => Prom
  */
 export interface Edit {
 	/**
-	 * Type contains the {@link EditType} of the edit being preformed
-	 * If not present it will assume the type is the type of the data
```

Comment:
We also still want this note about what it means for this to be missing.

Reply:
Thought I added it, will do it now

Reply:
I usually say something like "If not provided, ...". Empty usually implies something else (data that is present, but has nothing in it - empty object, empty array, etc.).